### ISSUE TITLE
searchTweets is python2

### ISSUE THREAD

**Issue by igordutra** [lucasfr/weatherFeelings#7]
We need to rewrite it to work with Python 3 and remove the unnecessary bits 
To be honest, since we won't be saving a csv and printing the results the way it suggests I think that's probably quite straightforward.

**Comment by lucasfr** [comment]
Yes, I will take care of it! Just need to organise myself. I am just afraid the twitter lib is written in Python 2 as well.

**Comment by igordutra** [comment]
We can try another lib, something like this - https://github.com/bear/python-twitter/

**Comment by igordutra** [comment]
@lucasfr Resolvi a bagaça - https://github.com/lucasfr/weatherFeelings/blob/master/searchTweets-python3.py I've changed the library and now it's compatible with Python 3 and much simpler to install. We now need to re-organise the code - it's a mess.
